### PR TITLE
tee-supplicant: Update TEEC_TEST_LOAD_PATH for Android

### PR DESCRIPTION
**Pull request by vchong**
Match TA_TEST_DIR in optee_test/blob/master/host/xtest/xtest_1000.c

Signed-off-by: Victor Chong <victor.chong@linaro.org>

### CODE REVIEW

File: tee-supplicant/Makefile [hunk 1]
```diff
@@ -45,7 +45,7 @@ TEES_CFLAGS	:= $(addprefix -I, $(TEES_INCLUDES)) $(CFLAGS) \
 ifeq ($(RPMB_EMU),1)
 TEES_CFLAGS	+= -DRPMB_EMU=1
 endif
-ifeq ($(CFG_TA_TEST_PATH),1)
```
Comment (etienne-lms) [inline tee-supplicant/Makefile]: should update also **tee-supplicant/tee_supplicant_android.mk**

Comment (vchong) [inline tee-supplicant/Makefile]: Good catch! Thanks!

**Comment by etienne-lms** [comment]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by vchong** [comment]
Squashed and tags added. Please merge if ok. Thanks!

**Pull request merged**